bitcoin/src/blockdata/transaction.rs

```diff
@@ -714,11 +714,16 @@ impl Transaction {
     #[deprecated(since = "0.31.0", note = "Use Transaction::stripped_size() instead")]
     pub fn strippedsize(&self) -> usize { Self::stripped_size(self).to_wu() as usize }
 
+    // TODO this should return a type that expresses size in terms of witness vs non-witness.
+    // struct Weight {
+    //     non_witness_bytes: u64,
+    //     witness_bytes: u64,
+    // }
```

Comment:
I don't like this being here, it should be an issue IMO since resolving it will take some design work.

Reply:
Good point. I created an issue here: https://github.com/rust-bitcoin/rust-bitcoin/issues/2059